target/linux/stm32/base-files/lib/preinit/83_switch_to_inband_interrupt_for_wifi_sdio

```diff
@@ -0,0 +1,9 @@
+
+. /lib/functions/system.sh
+
+switch_to_inband_interrupt_for_wifi_sdio() {
+	[ -e /lib/firmware/brcm/brcmfmac43430-sdio.txt ] && \
+		sed -i -E -e 's/^muxenab=0x10/#muxenab=0x10/' "$(readlink -f /lib/firmware/brcm/brcmfmac43430-sdio.txt)"
```

Comment:
I would prefer to have this at build time.
I think every board should have its unique nvram file.